tests/unit/index_tests.py

```diff
@@ -584,5 +585,134 @@ def test_delete_disabled(self):
             'Deleting the \"special\" index is not allowed.'
         )
 
+@unittest.skipUnless(
+    os.environ.get('RUN_CLOUDANT_TESTS') is not None,
```

Comment:
You've a double-negative here (unless/is not None). This would be simpler as `skipIf(os.environ.get('RUN_CLOUDANT_TESTS') is None,...`.